CIP-0001/README.md

```diff
@@ -12,180 +10,120 @@ License: CC-BY-4.0
-| * `Replaces`         | CIP number                                                  |
-| * `Superseded-By`    | CIP number                                                  |
-
+<details>
+  <summary><span id="licensing"><strong>Licensing</strong></span></summary>
 
-The Author(s) header lists the names and email addresses of the author or authors of the CIP.
-The format of the Author(s) header value must be:
+The following licenses are accepted. Each new CIP must identify at least one acceptable license in its preamble. In addition, each license must be referenced by its respective abbreviation below.
 
-  Random J. User \<address@dom.ain\>
+#### Recommended licenses
 
-If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions.
+- for software / code: Apache-2.0: [Apache License, version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+- for documentation: CC-BY-4.0: [Creative Commons Attribution 4.0 International Public License](https://creativecommons.org/licenses/by/4.0/legalcode)
 
-While a CIP is in private discussions (usually during the initial draft phase), a Discussions-To header will indicate the mailing list or URL where the CIP is being discussed. No Discussions-To header is necessary if the CIP is being discussed privately with the author.
+#### Unacceptable licenses
 
-The Type header specifies the type of CIP: Standards Track, Informational, or Process.
+All licenses not explicitly included in the above lists are not acceptable terms for a Cardano Improvement Proposal unless a later CIP extends this one to add them.
 
-The Created header records the date that the CIP was assigned a number, while Post-History is used to record when new versions of the CIP are posted to Cardano mailing lists.
-Dates should be in yyyy-mm-dd format, e.g. 2001-08-14.
+</details>
 
-CIPs may have a Requires header, indicating the CIP numbers that this CIP is dependent on.
+### Statuses
 
-CIPs may also have a Superseded-By header, indicating that a CIP has been rendered obsolete by a later document. The value is the number of the CIP that replaces the current document. The newer CIP must have a Replaces header containing the number of the CIP that it rendered obsolete.
+From its creation and onwards, a proposal evolves around the following statuses.
 
+Status   | Description
+---      | ---
+[Draft]    | An implicit status given to newly proposed CIPs that haven't yet been validated or reviewed.
+Proposed | Assigned to CIP that has been reviewed, accepted and now working towards acceptance. 'Proposed' generally means that implementation is ongoing or that adoption from various actors is missing before moving to 'Active'.
+Active   | The proposal has completed all steps needed for its activation. Said differently, it means observable metrics showing its adoption in the ecosystem.
+Obsolete | The CIP was either retired, made obsolete by a newer CIP or abandoned.
```

Comment:
What about CIPs where e.g. the implementation gets rejected? "Obsolete" doesn't feel like the right word to to cover all of "superseded", "retracted", "rejected", etc.

Reply:
There used to be a status "rejected" which I dropped here in favor of only Obsolete. That's perhaps not a wise decision.

Reply:
Maybe just "inactive"?